### DISCUSSION TITLE
Are there any drawbacks to disabling Webrtc and plugin scanning?

### DISCUSSION THREAD

**Discussion by FriedGenera** [yokoffing/Betterfox#349] [Q&A]
I had an old user.js that included both `media.peerconnection.enabled` and `plugin.scan.plid.all` set to false, do these still work or are they unnecessary? I read a long time ago that Webrtc can leak your IP and the second setting prevents sites from knowing what extensions you have, so I was wondering if I can just add them directly to the overrides section.

**Comment by yokoffing** [comment]
`plugin.scan.plid.all` is no longer a pref, AFAIK.

For WebRTC, you can read on the prefs but I wouldn't recommend changing them for practical purposes. If you never use video conferencing in Firefox, then you might be okay. https://github.com/yokoffing/Betterfox/blob/429abb61f52726b7c6f1675bccca32bbb5359c02/Securefox.js#L1050-L1078

**Reply by FriedGenera** [reply]
> `plugin.scan.plid.all` is no longer a pref, AFAIK.
> 
> For WebRTC, you can read on the prefs but I wouldn't recommend changing them for practical purposes. If you never use video conferencing in Firefox, then you might be okay.
> 
> https://github.com/yokoffing/Betterfox/blob/429abb61f52726b7c6f1675bccca32bbb5359c02/Securefox.js#L1050-L1078

Oh ok, is there a way to prevent websites from scanning your extensions then?
